create-release.py

```diff
@@ -610,6 +610,13 @@ def prepare_extensions_release() -> None:
             f"and execute them against local or cloud-based resources."
             f"See https://elyra.readthedocs.io/en/{config.new_version}/user_guide/enhanced-script-support.html",
         ),
+        "elyra-scala-editor-extension": SimpleNamespace(
+            packages=["scala-editor-extension", "metadata-extension", "theme-extension"],
+            description=f"The Scala Language editor extension contains support for Scala files, which can take "
+            f"advantage of the Hybrid Runtime Support enabling users to locally edit .scala files "
+            f"and execute them against local or cloud-based resources."
+            f"See https://elyra.readthedocs.io/en/{config.new_version}/user_guide/enhanced-script-support.html",
+        ),
```

Comment:
The extension also needs to be added to the publication list, like it is done for other components, e.g. https://github.com/elyra-ai/elyra/blob/6431c0f72b86bc7e5658ce94e889ab3415fc573c/create-release.py#L763-L764